src/Polar/kernels.jl

```diff
@@ -649,6 +649,8 @@ KA.@kernel function adj_branch_flow_edge_kernel!(
     Δθ = vang[fr_bus, j] - vang[to_bus, j]
     cosθ = cos(Δθ)
     sinθ = sin(Δθ)
+    dcosθ = -sin(Δθ)
```

Comment:
I am not sure about that, as `sin` is an expensive operation. How about doing instead?
```julia
    cosθ = cos(Δθ)
    sinθ = sin(Δθ)
    dcosθ = -sinθ
    dsinθ = cosθ
```

Reply:
Solely for debugging. This is not ready to merge in fact. Sorry for requesting review.

Reply:
Fixed.